frontend/src/components/errorPage/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Divider from '@material-ui/core/Divider';
+import style from './errorPage.module.scss';
+
+/**
+ * A basic error page.
+ */
+interface Props {
+  /** The error message. */
+  error: string;
+  /** Some more detail about the error. */
+  message: string;
+  /** OPTIONAL: Any extra information on a new line. */
+  extra?: string;
+}
+
+const ErrorPage = ({ error, message, extra }: Props) => (
+  <div className={style.fullPage}>
+    <div className={style.centered}>
+      <div>
+        <h1 className={style.header}>Error</h1>
+        <p>{error}</p>
+      </div>
+      <Divider orientation="vertical" flexItem />
+      <div>
+        <p>{message}</p>
+        {extra && (
+          <p>
+            <i>{extra}</i>
```

Comment:
Genuinely curious - what's the difference between the html `<i>` tag and CSS `font-style: italic;` ?

Reply:
I'm honestly not sure, however from what I can tell most browsers just have a default CSS rule of `font-style: italic;` for this element.